expoDemo/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import _ from 'lodash';
+import {screens} from 'unicorn-demo-app';
+import {NavigationContainer} from '@react-navigation/native';
+import {createStackNavigator} from '@react-navigation/stack';
+import MainScreen from './MainScreen';
+
+const Stack = createStackNavigator();
+
+export default function App() {
+  return (
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="Main">
+        <Stack.Screen name="Main" component={MainScreen}/>
+        {_.map(screens, (screen, key) => {
+          return <Stack.Screen key={key} name={key.replace('Screen', '')} component={screen}/>;
```

Comment:
Not sure why, but I'm not seeing anything except the title ("Main") (on iOS simulator).
Maybe we can look at it together.
BTW, I cannot connect with a real device (Android or iOS), but I'm guessing it's a routing issues.

Reply:
fixed